moveit_planners/ompl/ompl_interface/src/planning_context_manager.cpp

```diff
@@ -538,9 +538,10 @@ ModelBasedPlanningContextPtr PlanningContextManager::getPlanningContext(
 
   if (constrained_planning_iterator != pc->second.config.end() &&
       boost::lexical_cast<bool>(constrained_planning_iterator->second) &&
-      req.path_constraints.position_constraints.size() == 1)
+      (req.path_constraints.position_constraints.size() == 1 ||
```

Comment:
shouldn't this be xor?

Reply:
It should be xor, thanks for the catch. The new if statement looks somewhat unwieldy so I've also added a comment explaining the logic.